src/js/jsx/sections/libraries/Library.jsx

```diff
@@ -76,6 +76,15 @@ define(function (require, exports, module) {
          * @type {AdobeLibraryElement}
          */
         lastLocallyCreatedElement: null,
+        
+        /**
+         * Store the last locally updated graphic element and modified time. Used to determine whether the 
+         * library list should scroll to reveal the updated graphic.
+         * 
+         * @type {AdobeLibraryElement}
+         */
+        lastLocallyUpdatedGraphic: null,
```

Comment:
These look like they should be in the state, instead of open variables in the component. Is there a reason you didn't keep them in the state?